src/ServiceControl.Audit/Infrastructure/Settings/LoggingSettings.cs

```diff
@@ -25,8 +25,8 @@ public LoggingSettings(string serviceName, LogLevel defaultLevel = null, LogLeve
 
         LogLevel InitializeLevel(string key, LogLevel defaultLevel)
         {
-            var levelText = SettingsReader<string>.Read("ServiceControl.Audit", key);
```

Comment:
Ah so the `key` was used as a default value and `ServiceContro.Audit` as the name 🥇

Reply:
Yes, this was a real champion refactoring I did that introduced this bug. In all 3 SC instances too.